Vlad_Voitov/06.Ansible/ansible.cfg

```diff
@@ -0,0 +1,15 @@
+[defaults]
+inventory = ./hosts.ini
+interpreter_python = auto_silent
+forks = 5
+pipelining = True
+stdout_callback = ansible.posix.profile_tasks
+bin_ansible_callbacks = True
+host_key_checking = False
+
+[ssh_connection]
+pipelining = True
+ssh_args = -F /home/vlad/.ssh/config -o ControlMaster=auto -o ControlPersist=300s -o ServerAliveInterval=15 -o ServerAliveCountMax=3 -o IPQoS=throughput -o PreferredAuthentications=publickey -o IdentitiesOnly=yes -o IdentityFile=/home/vlad/.ssh/id_ed25519
```

Comment:
these settings are not recommended in this file, the are become environment depended. 
please pay attention to isolation, keep them with code (you have absolute path in config)

Reply:
it is just recommendation/advise